suricata.yaml.in

```diff
@@ -172,6 +172,7 @@ outputs:
             # http-body: yes           # enable dumping of http body in Base64
             # http-body-printable: yes # enable dumping of http body in printable format
             metadata: yes              # add L7/applayer fields, flowbit and other vars to the alert
+            # alert-metadata: yes      # enable dumping of key/value pairs defined by metadata keyword of signature
```

Comment:
'alert-metadata' is confusing. I would suggest 'rule-metadata' as it's the metadata from the rule